### PR TITLE
Merge [feature/phuongnt/read-football-player] to [main]

### PR DESCRIPTION
**Pull request by jason103nguyen**
Hi @trankhoa798, please help me review thi PR.
In this PR, I have created CRUD for football player entities.

### CODE REVIEW

File: src/main/java/com/fa/training/controller/FootballPlayerController.java [hunk 1]
```diff
@@ -19,30 +20,85 @@ public FootballPlayerController(FootballPlayerService footballPlayerService) {
...
+        System.out.println(fb_2);
+
+        return "hello";
+    }
+
+    @GetMapping(value = "/get-fb-id")
+    public String getFbById() throws DatabaseException {
+
+        FootballPlayerDto fb_1 = new FootballPlayerDto("FB_3", 1000,
+                LocalDate.parse("11-11-2021", DateTimeFormatter.ofPattern("dd-MM-yyyy")),
```
Comment (MaHai1993) [inline src/main/java/com/fa/training/controller/FootballPlayerController.java]: Move this `"dd-MM-yyyy"` to Constants file

File: src/main/java/com/fa/training/service/FootballPlayerServiceImpl.java [hunk 2]
```diff
@@ -20,17 +21,60 @@ public FootballPlayerServiceImpl(FootballPlayerDaoImpl footballPlayerDaoImpl) {
      * Add a football player to database
      * @param footballPlayerDto
      */
-    public void create(FootballPlayerDto footballPlayerDto) {
-        footballPlayerDaoImpl.create(new FootballPlayer(footballPlayerDto));
+    public int create(FootballPlayerDto footballPlayerDto) {
+
+        int id = (int) footballPlayerDaoImpl.create(new FootballPlayer(footballPlayerDto));
```
Comment (MaHai1993) [inline src/main/java/com/fa/training/service/FootballPlayerServiceImpl.java]: I think this line should be separated into two lines.
One is to create new FootballPlayer.
Second is footballPlayerDaoImpl.create(footballPlayer). 
Don't write code in one line.

**Review by MaHai1993 — CHANGES_REQUESTED** [review]
Fix these comments

File: src/main/java/com/fa/training/service/FootballPlayerServiceImpl.java (hunk 2 above)
Comment (jason103nguyen) [reply src/main/java/com/fa/training/service/FootballPlayerServiceImpl.java]: Thank you, I will fix this line.

File: src/main/java/com/fa/training/controller/FootballPlayerController.java (hunk 1 above)
Comment (jason103nguyen) [reply src/main/java/com/fa/training/controller/FootballPlayerController.java]: Thank you, I will fix this line.

**Pull request merged**